src/cpp/ext/proto_server_reflection.h

```diff
@@ -36,7 +36,11 @@
 #include <unordered_set>
 #include <vector>
 
+#ifdef GRPC_NO_GENERATED_CODE
```

Comment:
I see no reference to this define anywhere. Also, its behavior needs to be documented.